charts/pega/README.md

```diff
@@ -692,30 +692,60 @@ installer:
-`zero-downtime` |  If upgrading from 8.4.2 and later, use this option. A zero-downtime upgrade synchronizes the required update steps to minimize downtime and you and your customers can continue to access and use their applications in your environment with minimal disruption. This upgrade type places the rules into a read-only state, migrates the rules to your designated "new rules schema", migrates the data to your designated "temporary data schema", and then upgrades all of the rules to the new version. After upgrading your rules schema to the new rules, the process performs a rolling reboot of your nodes and then upgrades the data in the temporary data schema. Finally, the process redeploys the application using the new rules.
+`out-of-place` | `Deprecated for software upgrades`: Use this upgrade type only for zero-downtime patches. The process places the existing rules in your application into a read-only state, migrates the rules to your designated "new rules schema", migrates the data to your designated "temporary data schema", and then patches only the rules that were changed in the patch to the new version. With the new rules patched, it performs a rolling reboot, upgrades the data in the temporary data schema, and then redeploys your application using the new rules.
+`zero-downtime` |  If upgrading from 8.4.2 and later, use this option. This upgrade type places the rules into a read-only state, migrates the rules to your designated "new rules schema", migrates the data to your designated "temporary data schema", and then upgrades all of the rules to the new version. After upgrading your rules schema to the new rules, the process performs a rolling reboot of your nodes and then upgrades the data in the temporary data schema. Finally, the process redeploys the application using the new rules.
 `custom` |  Use this option for any upgrade in which you complete portions of the upgrade process in steps. Supported upgrade steps are: enable_cluster_upgrade rules_migration rules_upgrade data_upgrade disable_cluster_upgrade. To specify which steps to include in your custom upgrade, include them in your values.yaml file within the custom text. For example, to complete a rules upgrade without a data upgrade, you specify: installer.upgradeType: "custom" and installer.upgradeSteps: "enable_cluster_upgrade, rules_migration, rules_upgrade, disable_cluster_upgrade" custom upgrade can be used to configure upgrade in steps manner.
 `out-of-place-rules` | Use this option to complete an out-of-place upgrade of rules that the upgrade process migrates to the new rules schema. This schema will become the rules schema after your upgrade is complete.
 `out-of-place-data` |Use this option to complete an out-of-place upgrade of data the upgrade process migrates to a new, temporary data schema. The upgrade process removes this temporary schema after your application is running with updated data.
```

Comment:
We upgrade the data in the actual data schema

Reply:
thanks @arvasrikanth - how's this instead?
If upgrading from 8.4.2 and later, use this option. This upgrade type places the rules into a read-only state, migrates the rules to your designated "new rules schema", uses the temporary data schema to host some required data-specific tables, and then upgrades all of the rules to the new version. After upgrading your rules schema to the new rules, the process performs a rolling reboot of your nodes, upgrades any required data schema in the existing data schema, and then deletes the temporary data schema. Finally, the process redeploys the application using the new rules.

if out-of-place is deprecated, can we remove all of the wording for out-of-place and clarify that the "zero-downtime" process should be used for both patches and upgrades? @MadhuriArugula - thoughts?

Reply:
@taz-pega-work yes correct, we can use `zero-downtime` for both patches and upgrades. 

Also regarding this line from your comment - `upgrades any required data schema in the existing data schema, and then deletes the temporary data schema` 
Upgrade process doesn't delete temporary data schema. Can we update it to something like - `delete temporary data schema after the upgrade`